test/jdk/java/io/File/libGetXSpace.c

```diff
@@ -159,6 +159,32 @@ Java_GetXSpace_getSpace0
     (*env)->SetLongArrayRegion(env, sizes, 0, 4, array);
     return totalSpaceIsEstimated;
 }
+
+#ifdef WINDOWS
+JNIEXPORT jboolean JNICALL
+Java_GetXSpace_isCDDrive
```

Comment:
The native `GetXSpace` method needs to be defined on all platforms, otherwise we get the following on Unix:
```
java.lang.UnsatisfiedLinkError: 'boolean GetXSpace.isCDDrive(java.lang.String)'
	at GetXSpace.isCDDrive(Native Method)
	at GetXSpace$Space.<init>(GetXSpace.java:109)
	at GetXSpace.testVolumes(GetXSpace.java:353)
	at GetXSpace.main(GetXSpace.java:384)
```

Reply:
Sorry for my lack of consideration. Thank you for the proposal. I Fixed.

Reply:
Thanks. I verified the fix.